README.md

```diff
@@ -14,7 +14,7 @@ This capsule is the the companion code to the Bixby [Basic Cart Transactional Ca
 Buy a medium V-Neck
 Change the size
 (Select small)
-Make that a large V-Neck and add an Oxford Dress Shirt
```

Comment:
This was always a stretch but sad that we can no longer support this (it made a great demo). Do we know why this doesn't work anymore? We might be able to revisit when we make modeling changes and try Mark's idea on this: https://sixfivelabs.myjetbrains.com/youtrack/issue/VIV-27859

Reply:
I commented on that ticket. 
We can handle "make that a **large v-neck** and **add an oxford dress shirt**" but we cannot handle "add a v-neck and oxford dress shirt". Without any explanation, it's confusing for a template capsule, and we shouldn't keep unlearned training there while we wait for a remodel. We can add this example back once we are sure adding 2 items at the same time works.